apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -188,28 +189,13 @@ export class UpsertWorkflowUseCase {
       description: workflowDto.description || '',
       tags: workflowDto.tags || [],
       critical: false,
+      triggerIdentifier: `${slugify(workflowDto.name, {
```

Comment:
This introduces a new term called `triggerIdentifier`. Isn't this the same as the `workflowId`? If yes, please rename accordingly.

Reply:
Moreover we need to use the same help `slugifyIdentifier` helper

Reply:
> This introduces a new term called triggerIdentifier.

this is not true, under the hood CreateWorkflow usecase is responsible for creation of 
workflow.triggers.identifier
where workflow is NotificationTemplateEntity)
and triggers is NotificationTriggerEntity[]

In CreateWorkflow usecase context, i think we should provide it with more explicit terms if possible, until/if we decide to refactor our workflow entity